docs/reference/cli-commands/add.md

```diff
@@ -0,0 +1,22 @@
+(reference_pebble_add_command)=
+# add command
+
+The add command reads the plan's layer YAML from the path specified and
```

Comment:
Yes, the description is taken from the first paragraph below the usage in the help output.

Not all commands' short description fits in one line, I missed this and will fix it.

I did a comparison and found that the paragraph after the usage in the output is better than the short description in the `pebble help --all` command because for some commands it gives more useful information.